docs/core-concepts/atomic-swap-htlc.md

```diff
@@ -0,0 +1,146 @@
+---
+id: atomic-swap-htlc
+title: Atomic Swaps using HTLCs
+sidebar_label: Atomic Swaps using HTLCs
+---
+
+This documentation section provides an introduction to atomic swaps, hash time locked contracts (HTLCs) and how these concepts are used in COMIT.
+Atomic Swaps and HTLCs are core concepts of the COMIT protocol.
+
+<iframe 
```

Comment:
I think this whole slide should be after the 'Atomic Swap' and 'HTLC' subsections, since readers wouldn't know what the terms on the slide (e.g. HTLC) means before these subsections are read.

Reply:
The slide is a good attention grabber, that's why we put it there. It is not supposed to explain everything, but motivate the user to have a closer look on the content below.

I strongly vote for leaving it at the top, knowing that it does not answer all questions - other opinions? @bonomat @thomaseizinger